reconcile/elasticsearch_exceptions.py

```diff
@@ -0,0 +1,19 @@
+class ElasticSearchResourceNameInvalidError(Exception):
+    def __init__(self, msg):
+        super(ElasticSearchResourceNameInvalidError, self).__init__(
+            str(msg)
+        )
+
+
+class ElasticSearchResourceMissingSubnetIdError(Exception):
+    def __init__(self, msg):
+        super(ElasticSearchResourceMissingSubnetIdError, self).__init__(
+            str(msg)
+        )
+
+
+class ElasticSearchResourceVersionInvalidError(Exception):
+    def __init__(self, msg):
+        super(ElasticSearchResourceVersionInvalidError, self).__init__(
+            str(msg)
+        )
```

Comment:
I would prefer to have these exceptions under `utils` if that works

Reply:
👍